portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,13 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = randomFact;
 }
+
+
+function getFromServlet(){
+    fetch('/data').then(response => response.text()).then((saying) => { const sayingContainer = document.getElementById('saying-container');
+    console.assert(sayingContainer, "You have no element with ID 'saying-container'"); 
+    sayingContainer.innerHTML = saying;})
```

Comment:
Looks good!  Just add some whitespace to make this more readable.  Generally you want these to look like:
something.then(foo => {
  // code here...
});